addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -1,7 +1,10 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
+import re
 from odoo import api, fields, models, tools
 
+EMAIL_PATTERN = '([^ ,;<@]+@[^> ,;]+)'
```

Comment:
Isn't there a global regex used everywhere you can use? like in `from openerp.tools import email_re`

Reply:
If it does not exist I would effectively move it into tools/mail.py and import it here. Good remark @jem-odoo